src/main/java/org/web3j/protocol/core/personal/Personal.java

```diff
@@ -0,0 +1,33 @@
+package org.web3j.protocol.core.personal;
+
+import java.math.BigInteger;
+
+import org.web3j.protocol.Web3jService;
+import org.web3j.protocol.core.Request;
+import org.web3j.protocol.core.personal.methods.response.EcRecovered;
+import org.web3j.protocol.core.personal.methods.response.ImportedRawKey;
+import org.web3j.protocol.core.personal.methods.response.LockedAccount;
+import org.web3j.protocol.core.personal.methods.response.NewAccount;
+import org.web3j.protocol.core.personal.methods.response.Signed;
+import org.web3j.protocol.core.personal.methods.response.Transaction;
+import org.web3j.protocol.core.personal.methods.response.UnlockedAccount;
+import org.web3j.protocol.parity.methods.response.PersonalListAccounts;
+
+public interface Personal{
```

Comment:
My thinking is, those that are already in org.web3j.protocol.parity.Parity, but are common to Geth too, should be moved into a new Personal interface, i.e.:

- personal_listAccounts
- personal_newAccount
- personal_sendTransaction
- personal_unlockAccount

That way there's no duplication of code between the Parity & Geth specific client support in web3j.

The remainder that are exclusive to Parity should stay in org.web3j.protocol.parity.Parity. Then those that you've added which are exclusive to Geth should be in org.web3j.protocol.geth.Geth.